tests/modules/configs/test_config.py

```diff
@@ -0,0 +1,86 @@
+import os
+import pytest
+import sys
+
+from jsonschema import ValidationError
+from logging import Logger
+
+sys.path.append(os.getcwd())
+
+from src.modules.configs.config import ApplicationConf, ProjectConf, OperatorConf, \
+    MergedConf, validate_dict
+from src.modules.configs.schemas import application, project, operator
+from src.modules.global_vars import APPLICATION_DIR, CONFIG_RULES
+from tests.data.configs import validate_conf, err_msg, config_rule
+
+current_path = os.path.join(os.path.dirname(__file__))
+test_data_path = os.path.join(os.path.dirname(os.path.dirname(current_path)),
+                              "data", "configs")
+
+
+@pytest.mark.parametrize("conf, schema_file, err_msg", [
+    [validate_conf.application.get("unrecognized_key"), application,
```

Comment:
Each of those "getattrs" could be done in the function, would clear out the code.